.github/ISSUE_TEMPLATE/01-building-an-application.yml

```diff
@@ -14,8 +14,7 @@ body:
       description: In what type(s) of Android application(s) do you see this issue?
       multiple: true
       options:
-      - Classic Xamarin.Android (MonoAndroid13.0, etc.)
```

Comment:
I think these issue templates may need a little more thought.  We don't need to provide a dropdown with only one option.  I had planned to update this once .NET 7 was EOL later this month.

Reply:
Issue template changes reverted.